site/components/cart/CartItem/CartItem.tsx

```diff
@@ -86,29 +86,25 @@ const CartItem = ({
       <div className="flex flex-row space-x-4 py-4">
         <div className="w-16 h-16 bg-violet relative overflow-hidden cursor-pointer z-0">
           <Link href={`/product/${item.path}`}>
-            <a>
-              <Image
-                onClick={() => closeSidebarIfPresent()}
-                className={s.productImage}
-                width={150}
-                height={150}
-                src={item.variant.image?.url || placeholderImg}
-                alt={item.variant.image?.alt || 'Product Image'}
-                unoptimized
-              />
-            </a>
+            <Image
+              onClick={() => closeSidebarIfPresent()}
+              className={s.productImage}
+              width={150}
+              height={150}
+              src={item.variant.image?.url || placeholderImg}
+              alt={item.variant.image?.alt || 'Product Image'}
+              unoptimized
```

Comment:
I wonder if we should remove `unoptimized` here... hmm.

Reply:
It makes sense to be optimized here too. Changed to the real size, and also removed the zoom. It works with some demo PNGs here, but with other images looks cropped.